src/Avalonia.Themes.Fluent/DataValidationErrors.xaml

```diff
@@ -1,5 +1,16 @@
 <Style xmlns="https://github.com/avaloniaui" 
-       Selector="DataValidationErrors">
+       Selector="DataValidationErrors"
+       xmlns:sys="clr-namespace:System;assembly=netstandard">
+  <Design.PreviewWith>
+    <Border Padding="20"> 
+      <TextBox Text="Sample">
+        <DataValidationErrors.Error>
+          <sys:Exception/> 
+        </DataValidationErrors.Error>
```

Comment:
Just for case, FluentDark has Yellow color as error color.
![image](https://user-images.githubusercontent.com/3163374/88879519-7435d100-d1f8-11ea-94b6-1b3ed1faa8d9.png)